Source/Models/SwishError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Argo
+
+public enum SwishError {
+  case SessionError(NSError)
+  case ServerError(code: Int, json: AnyObject)
+  case JSONSerializationError(NSError)
+  case ArgoError(DecodeError)
+}
+
+extension SwishError: ErrorType { }
+
+public extension SwishError {
+  var rawError: NSError {
```

Comment:
I think that probably makes sense. We can then move from `ErrorType -> NSError` for the default `transformError` method. The only thing that wouldn't work for is the `.ServerError` case. Would you still transform that into an `NSError`?